django-backend/fecfiler/transactions/models.py

```diff
@@ -165,7 +165,24 @@ def children(self):
 
     @property
     def can_delete(self):
-        return len(self.blocking_reports) == 0
+        if len(self.blocking_reports) > 0:
+            return False
+        # For loan and debts, if there's a previous report you can't delete
+        if self.schedule_d or self.schedule_c:
```

Comment:
I feel like we should use similar logic here to whatever we decide below for determining loan/debt so we don't have inconsistencies.